.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Please fill out this information before submitting a bug:
+
+  - `ruby --version`: ...
+  - `gem --version`: ...
```

Comment:
the biggest problem is with PATH - it is the longest part of the output, maybe we could add an option to cut to first 4 entries and display "..."? - this would make the output cleaner leaving only the important information in the ticket